rest_framework/templates/rest_framework/base.html

```diff
@@ -79,38 +77,37 @@
           {% block request_forms %}
 
           {% if 'GET' in allowed_methods %}
-            <form id="get-form" class="pull-right">
+            <form id="get-form" class="float-end">
               <fieldset>
                 {% if api_settings.URL_FORMAT_OVERRIDE %}
-                  <div class="btn-group format-selection">
-                    <a class="btn btn-primary js-tooltip" href="{{ request.get_full_path }}" rel="nofollow" title="Make a GET request on the {{ name }} resource">GET</a>
-
-                    <button class="btn btn-primary dropdown-toggle js-tooltip" data-toggle="dropdown" title="Specify a format for the GET request">
-                      <span class="caret"></span>
+                  <div class="btn-group">
+                    <button class="btn btn-primary" data-bs-toggle="tooltip" href="{{ request.get_full_path }}" rel="nofollow" title="Make a GET request on the {{ name }} resource">GET</button>
+                    <button class="btn btn-primary dropdown-toggle dropdown-toggle-split" data-bs-toggle="dropdown">
+                      <span class="sr-only" data-bs-toggle="tooltip" title="Specify a format for the GET request">Toggle Dropdown</span>
```

Comment:
This renders "Toggle Dropdown" next to the dropdown control. I'm just going to remove this line for now.
If someone wants to add an `sr-only` here that doesn't render visibly, then let's tackle that separately.

```suggestion
```